x-pack/plugins/security_solution/public/management/pages/policy/view/host_isolation_exceptions/components/assign_flyout.tsx

```diff
@@ -228,6 +236,11 @@ export const PolicyHostIsolationExceptionsAssignFlyout = ({
     exceptionsRequest.isLoading,
   ]);
 
+  // do not render if doesn't have adecuate privleges
+  if (!privileges.loading && !privileges.canIsolateHost) {
```

Comment:
What about let this logic to the tab component? You will need to remove the query param and the flyout will not be displayed

Reply:
This is an extra precaution, in case someone decides to load this component somewhere else.